.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+env:
+  INFURA_API_KEY: ${{ secrets.INFURA_API_KEY }}
+  MNEMONIC: ${{ secrets.MNEMONIC }}
```

Comment:
Why is this needed? For testing?

Reply:
[hardhat.config.ts](https://github.com/coinvise/solidity-template/pull/1/files#diff-57cf5378f8cab20b02b279097ba6fad08eb53f747d81770045dda9c5f95effe3R22) uses the helper method [getEnvVar()](https://github.com/coinvise/solidity-template/pull/1/files#diff-36c97b5646983e5ed2835213316556fbb65223034c6fcb2deb035e6fa7a68610R1) which would fail if the env vars are not set

So none of the hardhat tasks/tests would work if those env vars are not set

It makes more sense in the local environment when we're deploying or testing against a particular network. But since it's being used in the `hardhat.config.ts`, the CI would also fail on `getEnvVar()`

Reply:
Can you make it not fail in the test environment? Not sure how safe it would be to store the Mnemonic

Reply:
The Mnemonic and infura api key doesnt actually have to be one that's being used / usable. It just have to present and non-empty. The infura key wont ever be used. Mnemonic would be used to generate the test accounts for the Hardhat network.
As long as we're not using the CI to do actual deployments, these values doesn't mean much, and just have to be non-empty and present in the env. So any non-empty string would just work. Is that alright?

Reply:
I think it's fine for now, we can improve it further down the line

Reply:
Actually, setting them in hardhat.config.ts as below works, although we wont be enforcing the env vars
```typescript
// Ensure that we have all the environment variables we need.
// const infuraApiKey: string = getEnvVar("INFURA_API_KEY");
// const mnemonic: string = getEnvVar("MNEMONIC");

const infuraApiKey: string = process.env.INFURA_API_KEY || "";
const mnemonic: string = process.env.MNEMONIC || "";
```